sympy/stats/tests/test_continuous_rv.py

```diff
@@ -333,6 +333,11 @@ def test_sample_continuous():
     sym, val = list(Z.pspace.sample().items())[0]
     assert sym == Z and val in Interval(0, oo)
 
+    samps = sample(Z, numsamples=3, seed=0)
+    assert round(next(samps), 4) == 0.7959
+    assert round(next(samps), 4) == 1.2559
+    assert round(next(samps), 4) == 0.9232
+
```

Comment:
Maybe add this to check all libraries, since it does not rely on hard-coded values (unsure if it fits in here)

```suggestion
    k = 2
    n = 10
    samples = dict(scipy=[], numpy=[], pymc3=[])
    for lib in samples.keys():
        for i in range(2):
            samples[lib].append(np.array(list(sample(Z, numsamples=n, library=lib, seed=0))))
    for lib, samps in samples.items():
        assert (
            all(
                [(samps[i] == samps[i + 1]).all() for i in range(k - 1)]
            ), 
            "Library " + lib + " samples differ despite seeding."
        )
    for lib in samples.keys():
        s0 = np.array(list(sample(Z, numsamples=n, library=lib, seed=0)))
        s1 = np.array(list(sample(Z, numsamples=n, library=lib, seed=1)))
        assert not (s0 == s1).all()
```